SR2023/2023-01-19-competition-venue-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+to: Student Robotics 2023 teams
+subject: SR2023 Competition Venue
```

Comment:
Maybe hint that there are other things too? Mostly so that they check this email when looking for the email which mentioned the tech day and simulator updates.

Reply:
Given how varied this email is, getting a good subject is going to be a nightmare

Reply:
Maybe?
> SR2023 Competition Venue, Tech Days & Simulator Update